tests/omprog-output-capture.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# This file is part of the rsyslog project, released under ASL 2.0
+
+# This test tests the 'output' setting of omprog when the feedback
+# feature is not used (confirmMessages=off).
+
+. $srcdir/diag.sh init
+generate_conf
+add_conf '
+module(load="../plugins/omprog/.libs/omprog")
+
+template(name="outfmt" type="string" string="%msg%\n")
+
+:msg, contains, "msgnum:" {
+    action(
+        type="omprog"
+	    binary=`echo $srcdir/testsuites/omprog-output-capture-bin.sh`
+        template="outfmt"
+        name="omprog_action"
+        output=`echo $RSYSLOG_OUT_LOG`
+        fileCreateMode="0644"  # default is 0600
+    )
+}
+'
+startup
+. $srcdir/diag.sh injectmsg 0 10
+shutdown_when_empty
+wait_shutdown
+
+expected_output="[stdout] Starting
```

Comment:
very minor: I have begin to call this variable ```EXPECTED```

Reply:
Changed together with the next one where applicable.